dspace/config/dspace.cfg

```diff
@@ -1596,6 +1596,23 @@ request.item.helpdesk.override = false
 # ${dspace.dir}/config/ directory.
 module_dir = modules
 
+####################################################################
+#------------------- SCOPUS SERVICES ------------------------------#
+#------------------------------------------------------------------#
+
+scopus.apiKey =
+# leave empty if you don't need to use an institutional token
+scopus.instToken =
+#The view mode to be used for the scopus search endpoint.
+#For more details see https://dev.elsevier.com/documentation/ScopusSearchAPI.wadl
+scopus.search-api.viewMode = COMPLETE
+
+####################################################################
+#------------------- WOS SERVICES ---------------------------------#
+#------------------------------------------------------------------#
+
+wos.apiKey =
```

Comment:
I would recommend we consider creating a new configuration file for *all* these new settings you will be creating in each of these new PRs.  Otherwise we'll be adding a ton to the `dspace.cfg`.   We also need to add comments to describe how to obtain an API key for each of these services.

Maybe we should either...
1. Create a new `*.cfg` for each external service (e.g. `scopus.cfg`, `wos.cfg`, etc)
2. OR, alternatively, we could just put these all in one `services-import.cfg` file. 

I think I lean slightly towards having a separate cfg per service (as this keeps them separate and easy to find). But, I'll leave it up to you and @abollini .  I just feel appending more to dspace.cfg might not be the best approach, as these are not general configurations.